webrender/src/prim_store.rs

```diff
@@ -532,8 +526,8 @@ impl ClipCorner {
 #[derive(Debug, Clone)]
 #[repr(C)]
 pub struct ImageMaskData {
-    uv_rect: DeviceRect,
-    local_rect: LayerRect,
+    pub local_rect: LayerRect,
+    pub padding: DeviceRect,
```

Comment:
It does at the moment - since the only old GPU stores we have left available is ```gpu_data32```. I intend to fix this up once we move the clip data to the GPU cache :)